kitsune/questions/views.py

```diff
@@ -680,6 +691,22 @@ def aaq(request, product_slug=None, step=1, is_loginless=False):
             content_type=user_ct,
         ).order_by("-id")[:IMG_LIMIT]
 
+        if form.is_bound and (topic := form.cleaned_data.get("category")):
+            # We've got invalid POST data, but the topic has been provided.
+            # Let's set the proper GA4 event parameters on the submit button.
+            context["submit_event_parameters"] = json.dumps(
+                {
+                    "topics": f"/{topic.slug}/",
```

Comment:
You could use a helper function to get the GA4 params in all 3 places that's needed.